app/services/git/syncs_updated_tracks.rb

```diff
@@ -0,0 +1,44 @@
+class Git::SyncsUpdatedTracks
+  QUIET_PERIOD = 10.seconds
+
+  def self.run
+    new.run
+  end
+
+  def self.sync
+    new.sync
+  end
+
+  def run
+    loop do
+      sync
+      puts "Sleeping"
```

Comment:
I need to get the rails logger working properly for rake tasks. It's a good idea to stick with PUTS for now, but I'm hoping to have that sorted today.